Maps/ArchipelagoCampaign/WoL/ap_gates_of_hell.SC2Map/MapScript.galaxy

```diff
@@ -3045,25 +3047,30 @@ bool gt_PeriodicRescueCheck_Func (bool testConds, bool runActions) {
                 if ((UnitGroupCount(UnitGroup(null, gv_p01_USER, RegionCircle(UnitGetPosition(auto8C24BA30_var), 2.5), UnitFilter(0, 0, (1 << c_targetFilterAir) | (1 << c_targetFilterStructure) | (1 << c_targetFilterMissile), (1 << (c_targetFilterDead - 32)) | (1 << (c_targetFilterHidden - 32))), 0), c_unitCountAlive) >= 1) && (UnitIsAlive(auto8C24BA30_var) == true)) {
                     libNtve_gf_AddUnitGroupToUnitGroup(gv_rescuableUnitGroup[lv_a][lv_b], lv_unitGroup);
                     UnitGroupClear(gv_rescuableUnitGroup[lv_a][lv_b]);
+                    break;
                 }
 
             }
         }
     }
     if ((UnitGroupCount(lv_unitGroup, c_unitCountAlive) >= 1)) {
         gv_statDropPodsRescued += 1;
-        if ((gv_statDropPodsRescued == 2)) {
+        if ((gv_statDropPodsRescued >= 2)) {
             lib5BD4895D_gf_AP_Core_sendLocationLocal(2);
         }
-        else if ((gv_statDropPodsRescued == 4)) {
+
+        if ((gv_statDropPodsRescued >= 4)) {
             lib5BD4895D_gf_AP_Core_sendLocationLocal(3);
         }
-        else if ((gv_statDropPodsRescued == 6)) {
+
+        if ((gv_statDropPodsRescued >= 6)) {
             lib5BD4895D_gf_AP_Core_sendLocationLocal(4);
         }
-        else if ((gv_statDropPodsRescued == 8)) {
+
+        if ((gv_statDropPodsRescued >= 8)) {

```

Comment:
Gonna revert this for now, just to get the more important stuff merged. I'm 90% sure this works now (actually testing it would require me to generate a whole bunch of worlds and manually unlock Gates of Hell in each of them, because I can't un-send a check) but until it's thoroughly tested I'll set it aside.